go/README.md

```diff
@@ -13,33 +13,33 @@ The following simple examples are included in this repository:
 
 -  `network_test`: Create and destroy a new external network.
 
--  `pool_test`: Join and eject host of Pool.
+-  `pool_test`: Add a server to a pool and then eject it.
 
--  `session_test`: Set up session over https protocal with or without certificate.
+-  `session_test`: Set up session over https protocol with or without certificate.
 
 -  `sr_nfs_test`: Creates a shared NFS SR.
 
 -  `sr_test`: Performs various base SR tests, including creating
     a dummy SR.
 
--  `vm_create_and_destroy_test`: Create and destroy a VM on the default SR with a network and DVD drive. Do same again with asynchronously call.
+-  `vm_create_and_destroy_test`: Create and destroy a VM on the default SR with a network and DVD drive. Repeat using asyncrhonous calls.
```

Comment:
asyncrhonous => asynchronous
Sorry this is my fault, it was a typo in my comment 😵